src/applications/hca/config/chapters/insuranceInformation/vaFacility_api.js

```diff
@@ -0,0 +1,92 @@
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import constants from 'vets-json-schema/dist/constants.json';
+import {
+  EssentialCoverageDescription,
+  FacilityLocatorDescription,
+} from '../../../components/FormDescriptions';
+import { ShortFormAlert } from '../../../components/FormAlerts';
+import VaMedicalCenter from '../../../components/FormFields/VaMedicalCenter';
+import {
+  emptyObjectSchema,
+  NotHighDisabilityOrNotCompensationTypeHigh,
+} from '../../../helpers';
+
+// define default schema properties
+const {
+  isEssentialAcaCoverage,
+  wantsInitialVaContact,
+} = fullSchemaHca.properties;
+
+// define states/territories with health care facilities
+const healthcareStates = constants.states.USA.filter(state => {
+  const statesWithoutService = ['AA', 'AE', 'AP', 'FM', 'MH', 'PW'];
```

Comment:
Should this get moved into the `constants` file?

Reply:
Good call!